src/CRUD.js

```diff
@@ -0,0 +1,33 @@
+import { Save } from './local-storage.js';
+
+const formInput = document.getElementById('formInput');
+
+export const create = (tasks, addAllTasksToHTML) => {
+  tasks.array.push({
+    completed: false,
+    description: formInput.value,
+    index: tasks.array.length,
+  });
+  formInput.value = '';
+  addAllTasksToHTML();
+  Save(tasks.array);
+};
+
+export const update = (tasks, index, value) => {
+  tasks.array[index].description = value;
+  Save(tasks.array);
+};
+
+export const Delete = (tasks, taskIndex, addAllTasksToHTML) => {
+  const tempTasks = [];
+  let index = 0;
+  for (let i = 0; i < tasks.array.length; i += 1) {
+    if (i !== taskIndex) {
+      tempTasks.push({ ...tasks.array[i], index });
+      index += 1;
+    }
+  }
+  tasks.array = tempTasks;
+  Save(tasks.array);
+  addAllTasksToHTML();
+};
```

Comment:
- I have this error when I try to add a task to the UI. Please fix this 🙏 
Your project:
<img width="960" alt="h4hw45h4" src="https://user-images.githubusercontent.com/73607512/220747503-b100a8d5-f69d-4638-baaf-7fcec3468e5f.PNG">

Reply:
It's working fine. I think you have to remove any previous data in local storage and it will work. Heres a recording.

https://user-images.githubusercontent.com/18634639/220755262-d788c3ad-8b2c-4a32-b3f7-9c9710f3b58b.mp4

Reply:
@mckent05 Please clear your local storage and try again